### ISSUE TITLE
EC2 InstanceTypeDef.Platform Literal incorrectly capitalized

### ISSUE THREAD

**Issue by andrew-womeldorf** [youtype/mypy_boto3_builder#245]
**Describe the bug**
EC2 `InstanceTypeDef` - the Platform field is typed `NotRequired[Literal["Windows"]]`, which is so close to being correct it hurts that it's not 😂 

- [AWS Docs](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Instance.html) indicate that it's `windows`, not `Windows`. This matches what we're receiving in practice (the lowercased version).

**To Reproduce**

1. Install `boto3-stubs[...]`
2. Run `mypy`/`pyright` on the following code sample

```python
from mypy_boto3_ec2.type_defs import InstanceTypeDef
instance: InstanceTypeDef = {"Platform": "windows"}
```

**Actual output**

```
$ poetry run pyright
<path>/main.py
  <path>/main.py:2:42 - error: Expression of type "dict[str, str]" cannot be assigned to declared type "InstanceTypeDef"
    "Literal['windows']" cannot be assigned to type "Literal['Windows']" (reportAssignmentType)
1 error, 0 warnings, 0 informations
```

**Additional context**
Your OS, `boto3-stubs` installation method, `boto3` version, etc.

boto3-stubs 1.34.38

**Comment by vemel** [comment]
Hello! Thank you for the report.

Looks like it is because of a mistake in `botocore` shapes. I will cross-check all dependent methods and stub this enum if it is safe to do.

**Comment by vemel** [comment]
`boto3` docs are also affected: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/run_instances.html#

Notice that `'Platform': 'Windows',`

**Comment by andrew-womeldorf** [comment]
Oooo, that's irritating. I wonder if it's worthwhile to accept either casing

**Comment by vemel** [comment]
I created a bug report: https://github.com/boto/botocore/issues/3128

No, looks like it is always `windows`, so I am going to temporarily stub this value until the bug is fixed.

**Comment by vemel** [comment]
@andrew-womeldorf fixed in `mypy-boto3-ec2 1.34.50`. Please update and let me know if it works as it should.

**Comment by andrew-womeldorf** [comment]
Fixed! Thank you very much!
